src/ContentHubLoggingClient.php

```diff
@@ -19,7 +19,7 @@ class ContentHubLoggingClient extends Client {
   use ContentHubClientTrait;
 
   // Override VERSION inherited from GuzzleHttp::ClientInterface.
-  const VERSION = '2.0.0';
+  const LIB_VERSION = '2.0.0';
```

Comment:
Hmm, wondering if we need that. The most appropriate solution would be to move the lib version and the user agent string to a separate, final class. Hence we'd only need to modify it at one place and we won't forget to change the version here too.
Could you please do that? We can call the class ContentHubDescriptor final class.